entities/author.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require_relative 'error'
+
+class Author # :nodoc:
+  attr_reader :name, :biography
+  def initialize(name, biography = nil)
+    raise LibraryError unless name.is_a? String
```

Comment:
- Why do you have the same error for 2 types of errors?
- Why you error have name LibraryError - do we have some trouble with Library? Try to name your variables/classes  more logic. Maybe WrongClassError or EmptyStringError or smth like that?
- Each class have validations. Maybe we create `Validator` class or module?
- Move your validations to a separate method in each class. For example `valid?`